Sources/web3swift/Web3/Web3+JSONRPC.swift

```diff
@@ -163,61 +161,61 @@ public struct JSONRPCresponse: Decodable{
         self.init(id: id, jsonrpc: jsonrpc, result: result, error: nil)
     }
 
-    /// Get the JSON RCP reponse value by deserializing it into some native <T> class.
+    /// Get the JSON RPC reponse value by deserializing it into some native <T> class.
     ///
     /// Returns nil if serialization fails
     public func getValue<T>() -> T? {
         let slf = T.self
         if slf == BigUInt.self {
-            guard let string = self.result as? String else {return nil}
-            guard let value = BigUInt(string.stripHexPrefix(), radix: 16) else {return nil}
+            guard let string = self.result as? String else { return nil }
+            guard let value = BigUInt(string.stripHexPrefix(), radix: 16) else { return nil }
             return value as? T
         } else if slf == BigInt.self {
-            guard let string = self.result as? String else {return nil}
-            guard let value = BigInt(string.stripHexPrefix(), radix: 16) else {return nil}
+            guard let string = self.result as? String else { return nil }
+            guard let value = BigInt(string.stripHexPrefix(), radix: 16) else { return nil }
             return value as? T
         } else if slf == Data.self {
-            guard let string = self.result as? String else {return nil}
-            guard let value = Data.fromHex(string) else {return nil}
+            guard let string = self.result as? String else { return nil }
+            guard let value = Data.fromHex(string) else { return nil }
             return value as? T
         } else if slf == EthereumAddress.self {
-            guard let string = self.result as? String else {return nil}
-            guard let value = EthereumAddress(string, ignoreChecksum: true) else {return nil}
+            guard let string = self.result as? String else { return nil }
+            guard let value = EthereumAddress(string, ignoreChecksum: true) else { return nil }
             return value as? T
         }
-//        else if slf == String.self {
-//            guard let value = self.result as? T else {return nil}
-//            return value
-//        } else if slf == Int.self {
-//            guard let value = self.result as? T else {return nil}
-//            return value
-//        }
+    //    else if slf == String.self {
+    //        guard let value = self.result as? T else { return nil }
+    //        return value
+    //    } else if slf == Int.self {
+    //        guard let value = self.result as? T else { return nil }
+    //        return value
+    //    }
```

Comment:
purposely left this block in, as these decodings might be of some utility down the road